src/layout/VerifyEmailWarning.js

```diff
@@ -0,0 +1,50 @@
+import { useDataQuery, useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { NoticeBox } from '@dhis2/ui'
+import React from 'react'
+
+const getUserProfileQuery = {
+    me: {
+        resource: 'me',
+    },
+}
+const getSystemSettingsQuery = {
+    systemSettings: {
+        resource: 'systemSettings',
+    },
+}
+
+const VerifyEmailWarning = () => {
+    const { data: userData, error: userError } =
+        useDataQuery(getUserProfileQuery)
+    const { data: systemData, error: systemError } = useDataQuery(
```

Comment:
I tried getting the `enforceVerifiedEmail` state from the `systemInfo` object directly from `useConfig` but couldn't, hence the api call.